ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/activityQuestion.component.ts

```diff
@@ -33,6 +35,7 @@ import { ActivityValidationResult } from '../../../models/activity/activityValid
 export class ActivityQuestionComponent implements OnInit, OnDestroy {
     @Input() block: ActivityQuestionBlock<any>;
     @Input() readonly: boolean;
+    @Input() layoutType: LayoutType = LayoutType.DEFAULT;
```

Comment:
That default is DEFAULT. Have this in a lot or all the the answer/question components. Get used to seeing it!